SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMigration/SpatialTestCharacterMigration.cpp

```diff
@@ -8,6 +8,17 @@
 #include "SpatialFunctionalTestFlowController.h"
 #include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/TestMovementCharacter.h"
 
+namespace
+{
+float GetTargetDistanceOnLine(const FVector& From, const FVector& Target, const FVector& Location)
```

Comment:
In this simple test we are only moving the character forward and backward along the x-axis so we can simplify this and just test the character.position.x against an x position instead of a trigger box.

Reply:
Just and idea but I don't know if we need to simplify this any more. This is not exactly complex logic and isn't going to impact performance. If the test ever changes to moving in the y, the test would break and someone would have to work out why.

Reply:
Good point, I'm happy to leave it with the generic solution.